### PR TITLE
fix SS_E08 gradient gravity again

### PR DESCRIPTION
**Pull request by youbitchoc**
needs testing

### CODE REVIEW

**Comment by youbitchoc** [comment]
previous was working like
```C#
if (room.physicalObjects[i][j] is Player || self.room.physicalObjects[i][j].abstractPhysicalObject.IsLocal())
```
rather than `&&`

**Review by 6fears7 — APPROVED** [review]
I have no idea if you’re waiting for approvals or just committing to pull in later but LGTM once proven tested

**Pull request merged**
